docs-source/content/userguide/experimental/istio.md

```diff
@@ -68,16 +64,10 @@ $ kubectl create namespace domain1
 $ kubectl label namespace domain1 istio-injection=enabled
 ```
 
-Currently, the experimental Istio support is provided only for domains stored on
-persistent volumes.  To enable the support for a domain, you need to add the
-`experimental` section to your domain custom resource YAML file as shown in the
+To enable the support for a domain, you need to add the
+`configuration` section to your domain custom resource YAML file as shown in the
```

Comment:
as shown in the example below -> as shown in the following example:   (that is, if it is the following example; I couldn't quite tell.)